Common/Securities/SecurityService.cs

```diff
@@ -78,7 +78,13 @@ public class SecurityService : ISecurityService
                 throw new ArgumentException($"Symbol can't be found in the Symbol Properties Database: {symbol.Value}");
             }
 
-            var symbolProperties = _symbolPropertiesDatabase.GetSymbolProperties(symbol.ID.Market, symbol, symbol.ID.SecurityType, defaultQuoteCurrency);
+            // For Futures Options that don't have a SPDB entry, the futures entry will be used instead.
```

Comment:
nit - comment is off

Reply:
This still remains true for those with no entries. The list of tickers we will support have some that rely on this fall-through behavior.